snafu-derive/src/lib.rs

```diff
@@ -919,10 +904,9 @@ impl syn::parse::Parse for Source {
     }
 }
 
-enum Backtrace {
-    Flag(bool),
-    Delegate,
-}
+// Having a Backtrace newtype and implementing Parse gives us a way to handle careful parsing
+// errors outside of `impl Parse for SnafuAttribute`
```

Comment:
I really appreciate that you've been adding documentation; I tend to be pretty lazy for the "internal" docs.